src/dotnet/commands/dotnet-restore/LocalizableStrings.resx

```diff
@@ -169,4 +169,7 @@
     <value>Force all dependencies to be resolved even if the last restore was successful.
 This is equivalent to deleting project.assets.json.</value>
   </data>
+  <data name="CmdInteractiveRestoreOptionDescription" xml:space="preserve">
+    <value>Allow restore to block and require manual action to proceed.</value>
```

Comment:
If the previous message is reworded, this one should be reworded too.

Reply:
I would suggest exactly the same message - but I'm assuming they are both --interactive options.

Reply:
Discussed with Kathleen, let's use the same message for both (ie it's OK to say "the command" instead of "restore") in both cases.